deploy/kabanero-customresources.yaml

```diff
@@ -556,3 +556,22 @@ subjects:
 - kind: ServiceAccount
   name: event-listener
   namespace: kabanero
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kabanero-cli-service-deployments-role
+  namespace: kabanero
+  labels:
+    kabanero.io/install: 27-cli-service-role
```

Comment:
We'll need to change the install.sh and uninstall.sh scripts to create and remove this new resource.
Example from install.sh: https://github.com/kabanero-io/kabanero-operator/blob/4ad97f5445ec20d7965e5bb97252165f9cc6e025/deploy/install.sh#L341
Example from uninstall.sh: https://github.com/kabanero-io/kabanero-operator/blob/4ad97f5445ec20d7965e5bb97252165f9cc6e025/deploy/uninstall.sh#L119